wagtail/wagtailcore/tests/test_blocks.py

```diff
@@ -106,21 +107,57 @@ def get_prep_value(self, native_value):
         self.assertEqual('hello world', value_from_form)
 
 
-class TestRichTextBlock(unittest.TestCase):
+class TestRichTextBlock(TestCase):
+    fixtures = ['test.json']
```

Comment:
Yes - it's used when testing that links are correctly expanded from `<a linktype="page" id="4">Christmas</a>` to `<a href="/events/christmas/">Christmas</a>`.